src/univariate/discrete/negativebinomial3.jl

```diff
@@ -0,0 +1,134 @@
+```math
+n \\sim \\text{Poisson}(z), \\quad
+z \\sim \\text{Gamma}(\\alpha, \\beta)
+```
+
+```julia
+NegativeBinomial3()        # distribution with α = 1.0 and β = 1.0
+NegativeBinomial3(α, β)    # distribution with shape α and scale β
+
+convert(NegativeBinomial{T}, d)         # Parametric conversion to NegativeBinomial
+convert(NegativeBinomial2Log{T}, d)     # Parametric conversion to NegativeBinomial2Log
+convert(NegativeBinomial3{T}, d)        # Parametric conversion to NegativeBinomial3
+```
+
+External links:
+* [Negative binomial distribution, Bayesian Data Analysis (3rd edition), Appendix A](http://www.stat.columbia.edu/~gelman/book/BDA3.pdf)
+Note: the definition above uses the scale parameterization of the Gamma distribution
+(matching Distributions.jl), whereas BDA uses the inverse scale parameterization.
+
+See also: [`NegativeBinomial2`](@ref), [`NegativeBinomial2Log`](@ref)
+"""
+struct NegativeBinomial3{T<:Real} <: DiscreteUnivariateDistribution
+    α::T
+    β::T
+    NegativeBinomial3{T}(α::T, β::T) where {T<:Real} = new{T}(α, β)
+end
+
+function NegativeBinomial3(α::T, β::T; check_args::Bool=true) where {T<:Real}
+    @check_args NegativeBinomial3 (α, α > zero(α)) (β, β > zero(β))
+    return NegativeBinomial3{T}(α, β)
+end
+
+NegativeBinomial3(α::Real, β::Real; check_args::Bool=true) = NegativeBinomial3(promote(α, β)...; check_args=check_args)
+NegativeBinomial3(α::Integer, β::Integer; check_args::Bool=true) = NegativeBinomial3(float(α), float(β); check_args=check_args)
+NegativeBinomial3(α::Real; check_args::Bool=true) = NegativeBinomial3(α, 1.0; check_args=check_args)
+NegativeBinomial3() = NegativeBinomial3{Float64}(1.0, 1.0)
+
+@distr_support NegativeBinomial3 0 Inf
+
+insupport(d::NegativeBinomial3, x::Real) = false
+insupport(d::NegativeBinomial3, x::T) where {T<:Integer} = x ≥ 0
+#### Conversions
+
+function convert(::Type{NegativeBinomial3{T}}, α::Real, β::Real) where {T<:Real}
+    NegativeBinomial3(T(α), T(β))
+end
+function convert(::Type{NegativeBinomial3{T}}, d::NegativeBinomial3) where {T<:Real}
+    NegativeBinomial3{T}(T(d.α), T(d.β))
+end
+convert(::Type{NegativeBinomial3{T}}, d::NegativeBinomial3{T}) where {T<:Real} = d
+
+# # Interconversion
+# function convert(::Type{NegativeBinomial3{T}}, d::NegativeBinomial) where {T<:Real}
+#     NegativeBinomial3{T}(T(d.r), T((1 - d.p) / d.p))
+# end
+# function convert(::Type{NegativeBinomial3{T}}, d::NegativeBinomial2) where {T<:Real}
+#     NegativeBinomial3{T}(T(d.ϕ), T(d.μ / d.ϕ))
+# end
+# function convert(::Type{NegativeBinomial3{T}}, d::NegativeBinomial2Log) where {T<:Real}
+#     NegativeBinomial3{T}(T(d.ϕ), T(exp(d.η) / d.ϕ))
+# end
+
+#### Parameters
+
+params(d::NegativeBinomial3) = (d.α, d.β)
+partype(::NegativeBinomial3{T}) where {T} = T
+
+succprob(d::NegativeBinomial3{T}) where {T} = inv(d.β + one(T))
+failprob(d::NegativeBinomial3{T}) where {T} = (β = d.β; β / (β + one(T)))
+
+#### Statistics
+
+mean(d::NegativeBinomial3{T}) where {T} = d.α * d.β
```

Comment:
```suggestion
mean(d::NegativeBinomial3) = d.α * d.β
```

Reply:
It's true, this `{T} where {T}` should not be here.